src/compressed_tensors/quantization/lifecycle/forward.py

```diff
@@ -257,13 +257,10 @@ def _process_quantization(
         QuantizationStrategy.GROUP,
         QuantizationStrategy.TENSOR_GROUP,
     ):
-        n_dims = x.shape
-        if len(n_dims) > 2:
-            x = x.squeeze(0)
 
         output_dtype = dtype if dtype is not None else x.dtype
         output = torch.zeros_like(x).to(output_dtype)
-        columns = output.shape[1]
+        columns = output.shape[-1]
```

Comment:
moving to last dim makes sense to me, should confirm with rest of team though that this will work fine with `TENSOR_GROUP` or any other edge cases

Reply:
I'm not quite sure What's going on with the diff, but this is unrelated